composer.json

```diff
@@ -11,15 +11,15 @@
     ],
     "license" : "MIT",
     "require": {
-        "illuminate/support": "^5.8",
-        "illuminate/container": "^5.8",
-        "illuminate/database": "^5.8",
-        "illuminate/events": "^5.8",
+        "illuminate/support": "^6.0",

```

Comment:
Are there any super breaking changes that would make the package only support the latest version? Can we keep compatibility with 5.8?

Reply:
Could this fix be put with a new 3.x release, as it seems if it requires the the ^5.8 branches it seems the composer install fails as it doesn't have the same versions?

Reply:
Why would it fail?

Reply:
Try `"^5.8|^6.0"`